src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -5,8 +5,8 @@
     Code coverage package versions go here and in the test-runtime-packages.config
   -->
   <PropertyGroup>
-    <OpenCoverVersion>4.6.476-rc</OpenCoverVersion>
-    <ReportGeneratorVersion>2.4.0</ReportGeneratorVersion>
+    <OpenCoverVersion>4.6.519</OpenCoverVersion>
```

Comment:
Sure, sounds fine.  I've simply been following what I thought was the desired approach.  Happy to change to something that doesn't require touching buildtools.